resources/Microsoft.VSCode.DSC/Microsoft.VSCode.DSC.psm1

```diff
@@ -0,0 +1,125 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+$ErrorActionPreference = "Stop"
+Set-StrictMode -Version Latest
+
+enum Ensure
```

Comment:
Enums and classes are not namespaced when loaded into PowerShell; you should use a more specific name like `VSCodeEnsure`.

Reply:
Changed to `VSCodeEnsure`